extensions/WEBGL_color_buffer_float/extension.xml

```diff
@@ -65,6 +76,7 @@
 [NoInterfaceObject]
 interface WEBGL_color_buffer_float {
   const GLenum RGBA32F_EXT = 0x8814;
+  const GLenum RGB32F_EXT = 0x8815;
```

Comment:
Don't add this, at least not in this bug. Supporting renderbuffers here is scope creep.

Reply:
Removed